bench/basic-app/pages/api/index.js

```diff
@@ -1,3 +0,0 @@
-export default function handler(req, res) {
```

Comment:
Would be good to keep these but prefix them with `page-routes` or something

Reply:
Do the API endpoints get picked up from within the `pages/` directory if the `app/api` exists?

Reply:
yeah as long as sub-path differs e.g. `pages/api/page-api.js`

Reply:
Ah okay I must've been spelling something wrong. Works now. Restoring with a new name.